src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -1630,9 +1641,13 @@ module Util =
         | Fable.NewTuple (values, isStruct) -> makeTuple com ctx r isStruct values
         | Fable.NewList (headAndTail, typ) -> makeList com ctx r typ headAndTail
         | Fable.NewOption (value, typ, isStruct) -> makeOption com ctx r typ value isStruct
-        | Fable.NewRecord (values, entRef, genArgs) -> makeRecord com ctx r values entRef genArgs
```

Comment:
@alexswan10k Please remove change as `isCopyableEntity` already handles value types correctly.

Reply:
This is fine, I can remove it later.